src/ui/MainWindow.cc

```diff
@@ -183,8 +183,10 @@ MainWindow::MainWindow(QSplashScreen* splashScreen)
     // Qt 4 on Ubuntu does place the native menubar correctly so on Linux we revert back to in-window menu bar.
     // TODO: Check that this is still necessary on Qt5 on Ubuntu
 #ifdef Q_OS_LINUX
+#ifndef __android__
     menuBar()->setNativeMenuBar(false);
 #endif
+#endif
     

```

Comment:
There isn't really much in the menu. I think we can get rid of it completely and just include a hamburger with a few things on it.